LogToDB.cs

```diff
@@ -14,6 +15,61 @@ private string Logging(string type, string message, string username = "")
             var sql = $"INSERT INTO log VALUES ('{DateTime.Now}', '{type}', '{username}', '{message}')";
             return sql;
         }
+
+        public async Task<string> AsyncLog(string type, string username, string message)
+        {
+            return await Task.Run(() =>
+            {
+                Log(type, username, message);
```

Comment:
```suggestion
                LogAsync(type, username, message);
```
Почему не асинхронный метод?